v3/internal/parser/models_test.go

```diff
@@ -0,0 +1,74 @@
+package parser
+
+import (
+	"github.com/google/go-cmp/cmp"
+	"strings"
+	"testing"
+)
+
+const expected = `
+  export class Person {
```

Comment:
If we surrounded this with a namespace called "main", would that fix the self reference?